jenkins/release.jenkinsFile

```diff
@@ -0,0 +1,16 @@
+lib = library(identifier: 'jenkins@1.5.3', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+standardReleasePipelineWithGenericTrigger(
+    tokenIdCredential: 'jenkins-sql-odbc-generic-webhook-token',
+    causeString: 'A tag was cut on opensearch-project/sql-odbc repository causing this workflow to run',
+    downloadReleaseAsset: true,
+    publishRelease: true) {
+        publishToMaven(
+            signingArtifactsPath: "$WORKSPACE/repository/",
+            mavenArtifactsPath: "$WORKSPACE/repository/",
+            autoPublish: true
+        )
```

Comment:
I believe we want to publish to website so this should be replaced by [publishToArtifactsProdBucket](https://github.com/opensearch-project/opensearch-build-libraries/blob/main/vars/publishToArtifactsProdBucket.groovy). 
```suggestion
                        publishToArtifactsProdBucket(
                            assumedRoleName: 'sql-odbc-upload-role',
                            source: 'opensearch-sql-odbc-driver-32-bit-1.5.0.0-Windows.msi',
                            destination: 'opensearch-clients/odbc/opensearch-sql-odbc-driver-32-bit-1.5.0.0-Windows.msi',
                            signingPlatform: 'windows',
                            sigType: 'null',
                            sigOverwrite: true
                        )
                     publishToArtifactsProdBucket(
                            assumedRoleName: 'sql-odbc-upload-role',
                            source: 'opensearch-sql-odbc-driver-64-bit-1.5.0.0-Windows.msi',
                            destination: 'opensearch-clients/odbc/opensearch-sql-odbc-driver-64-bit-1.5.0.0-Windows.msi',
                            signingPlatform: 'windows',
                            sigType: 'null',
                            sigOverwrite: true
                        )
                      publishToArtifactsProdBucket(
                            assumedRoleName: 'sql-odbc-upload-role',
                            source: 'opensearch-sql-odbc-driver-64-bit-1.5.0.0-Darwin.pkg,
                            destination: 'opensearch-clients/odbc/opensearch-sql-odbc-driver-64-bit-1.5.0.0-Darwin.pkg',
                            signingPlatform: 'macos',
                            sigType: 'null',
                            sigOverwrite: true
                        )
```